lib/Dialect/TritonGPU/Transforms/Pipeliner/PipeliningUtility.cpp

```diff
@@ -248,19 +248,24 @@ int mlir::triton::getCopyVecBytes(RankedTensorType registerTy,
 
 void mlir::triton::serializeLatencies(ModuleOp module,
                                       DenseMap<Operation *, int> &opLatency) {
+  auto helper = module.getContext()
+                    ->getLoadedDialect<TritonDialect>()
+                    ->getLatencyAttrHelper();
+  auto builder = Builder(module);
   for (auto &[op, latency] : opLatency) {
-    op->setAttr(
-        kLatencyAttrName,
-        IntegerAttr::get(IntegerType::get(module.getContext(), 32), latency));
+    helper.setAttr(op, builder.getI32IntegerAttr(latency));
   }
 }
 
 DenseMap<Operation *, int> mlir::triton::deserializeLatencies(Operation *op) {
+  auto helper = op->getContext()
```

Comment:
suggestion: can you add a shorthand for this in Triton/Dialect.h?

Reply:
Yes! added.

Reply:
posted these upstream..

Reply:
well.. I guess Medhi and River are not big fans..

Reply:
They tend to be picky :P